src/Microsoft.ServiceFabric.Client/IApplicationClient.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.ServiceFabric.Client
     /// <summary>
     /// Interface containing methods for performing ApplicationClient operataions.
     /// </summary>
-    public partial interface IApplicationClient
```

Comment:
Keep it as partial this interface is auto-generated from swagger specification, which generates all interfaces and classes as partial to allow manual extension later.